libraries/AP_RangeFinder/AP_RangeFinder_MAVLink.cpp

```diff
@@ -69,7 +69,7 @@ void AP_RangeFinder_MAVLink::update(void)
         set_status(RangeFinder::RangeFinder_NoData);
         state.distance_cm = 0;
     } else {
-        set_status(RangeFinder::RangeFinder_Good);
         state.distance_cm = distance_cm;
+        update_status();
```

Comment:
Yes, that's why I did it. The Min/max parameters for the rangefinder were being ignored.